### ISSUE TITLE
Internal shape error when asserting coords.ndim == 3

### ISSUE THREAD

**Issue by gph82** [mdtraj/mdtraj#1234]
Hi! 
I'm not really sure of whats going on here, but using the "atom_indices" slice when loading an xtc is failing like this:
```

In [48]: md.version.version
Out[48]: '1.8.0'
In [49]: traj = md.load('frames0-116.Prot-Masses_PLM.xtc', top='../../3CAP.Prot-Masses_PLM.npt.eq.pdb', stride=50)
In [50]: traj.n_atoms
Out[50]: 5226
In [51]: selection = traj.top.select('protein and not element  H')
In [52]: len(selection), selection.max()
Out[52]: (2621, 5191)
In [53]: traj = md.load('frames0-116.Prot-Masses_PLM.xtc', top='../../3CAP.Prot-Masses_PLM.npt.eq.pdb', stride=50, atom_indices=selection)
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-53-e86b50d8b420> in <module>()
----> 1 traj = md.load('frames0-116.Prot-Masses_PLM.xtc', top='../../3CAP.Prot-Masses_PLM.npt.eq.pdb', stride=50, atom_indices=selection)

/home/mi/gph82/miniconda3/lib/python3.4/site-packages/mdtraj/core/trajectory.py in load(filename_or_filenames, discard_overlapping_frames, **kwargs)
    370         topkwargs = kwargs.copy()
    371         topkwargs.pop("top", None)
--> 372         kwargs["top"] = _parse_topology(kwargs["top"], **topkwargs)
    373 
    374     # grab the extension of the filename

/home/mi/gph82/miniconda3/lib/python3.4/site-packages/mdtraj/core/trajectory.py in _parse_topology(top, **kwargs)
    167 
    168     if isinstance(top, string_types) and (ext in ['.pdb', '.pdb.gz', '.h5','.lh5']):
--> 169         _traj = load_frame(top, 0, **kwargs)
    170         topology = _traj.topology
    171     elif isinstance(top, string_types) and (ext in ['.prmtop', '.parm7']):

/home/mi/gph82/miniconda3/lib/python3.4/site-packages/mdtraj/core/trajectory.py in load_frame(filename, index, top, atom_indices, **kwargs)
    309         _assert_files_or_dirs_exist(filename)
    310 
--> 311     return loader(filename, frame=index, **kwargs)
    312 
    313 

/home/mi/gph82/miniconda3/lib/python3.4/site-packages/mdtraj/formats/pdb/pdbfile.py in load_pdb(filename, stride, atom_indices, frame, no_boxchk, standard_names)
    157         else:
    158             coords = f.positions[::stride, atom_slice, :]
--> 159         assert coords.ndim == 3, 'internal shape error'
    160         n_frames = len(coords)
    161 

AssertionError: internal shape error
```

**Comment by ctk3b** [comment]
Can you try pulling the latest source? I think this is fixed by #1227

**Comment by gph82** [comment]
sorry, still there with:
In [4]: md.version.full_version
Out[4]: '1.8.0.dev0+20341059fe6d5d343259bc4bd8f9821397924800'

**Comment by mpharrigan** [comment]
cc https://github.com/mdtraj/mdtraj/issues/1212 ?

**Comment by smsaladi** [comment]
Also facing this issue

**Comment by mpharrigan** [comment]
Are you sure this isn't fixed by #1227 ?

**Comment by mpharrigan** [comment]
try this with the new release of mdtraj 1.9 and re-open if the issue is not fixed
